dashboard/agent.py

```diff
@@ -185,8 +185,9 @@ async def _check_parent():
                 agent_port=self.grpc_port,
                 agent_ip_address=self.ip))
 
-        await asyncio.gather(check_parent_task,
-                             *(m.run(self.server) for m in modules))
+        if sys.platform not in ["win32", "cygwin"]:
+            await asyncio.gather(check_parent_task,
+                                 *(m.run(self.server) for m in modules))
```

Comment:
If we really don't need the `check_parent_task` on Windows, then we can:
```python
concurrent_tasks = [m.run(self.server) for m in modules]
if sys.platform not in ["win32", "cygwin"]:
    concurrent_tasks.append(check_parent_task)
await asyncio.gather(*concurrent_tasks)
```
This fix is a walkaround for that we can't find out why passing environment var from raylet to Python worker process fail on Windows. After this PR is merged, the dashboard agent process may be leaking, e.g. user kills the raylet, but the dashboard agent is still alive.

Reply:
This PR hides the root cause and makes the error gone.

Reply:
Good point, I missed that second line of the gather! I'll fix that.